e2e/playwright/payUserFlow.spec.ts

```diff
@@ -0,0 +1,276 @@
+import { test, expect, type Page } from "@playwright/test";
+import { readFileSync } from "fs";
+import path from "path";
+import { SUCCESS_CARD_NUMBER } from "./constants";
+import { awaitForPaymentResponse } from "./payUtils/awaitForPaymentResponse";
+import { fillGovUkCardDetails } from "./payUtils/fillGovUkCardDetails";
+import {
+  createFlow,
+  createTeam,
+  deleteTestData,
+  getAnalyticsByFlowId,
+  gqlAdmin,
+  insertPublishedFlow,
+  insertTestUser,
+  prodGqlPublic,
+} from "./utils";
+
+test.setTimeout(90_000);
+
+const URL = "http://localhost:3000";
+
+const TEST_EMAIL = "test@test.com";
+const DEFAULT_WAIT_TIME = 300;
+const TEAM_SLUG = "buckinghamshire";
+const FLOW_SLUG = "apply-for-a-lawful-development-certificate";
```

Comment:
a couple nits here:
- are any of these consts perhaps a better fit for `e2e/playwright/constants.ts` ? as I understand it, only team & flow slugs should be specific to test file but perhaps email, wait time and URL could be more explicitly shared? 
- can we please pull the LDC flow from a team that is currently live - eg Southwark or Lambeth - for a better representation of / confidence in "live" service content that is being actively maintained?